Drakhtar/Battalion.cpp

```diff
@@ -1,14 +1,41 @@
 // Copyright 2019 the Drakhtar authors. All rights reserved. MIT license.
 
 #include "Battalion.h"
+#include "Game.h"
+
+
+Battalion::Battalion(Texture * texture, Box * box, int attack, int health,
+int speed, int attackRange, int moveRange, int battalionSize)
+    : Unit(texture, box, attack, health, speed, attackRange, moveRange),
+    battalionSize_(battalionSize)
+    {
+    health_ = Unit::getMaxHealth() * battalionSize_;
+
+    SDL_Color textColor;
+    textColor.r = 255;
+    textColor.g = 255;
+    textColor.b = 255;
+    textColor.a = 255;
+    healthText->setText("Salud: " + to_string(this->getHealth()));
+
+    sizeText_ =
```

Comment:
This would look much better if written this way:

```cpp
    auto rect = box_->getRect();
    sizeText_ = new Text(Game::getInstance()->getRenderer(), 
                         FontManager::get("Retron2000"),
                         { rect.x + rect.w / 2, rect.y + rect.h * 4 / 5 },
                         textColor, 
                         "Units: " + to_string(battalionSize_),
                         box_->getRect().w * 2);
```